file-server/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY . .
 COPY fs_config.template.json fs_config.json
 
 RUN mkdir /app/build
-COPY --from=admin         /app/build/ /app/build
+COPY --from=admin         /app/dist/  /app/build
```

Comment:
@patcon Let's please keep the `app/build` path. I realize the way things were set up was kind of weird in that by default things were going to `dist`, unless you specified `build` in the `LOCAL_OUPUT_PATH`, but we should stick with the ultimate paths which were in use at the Docker level (`app/build`).

Reply:
Sounds good, should be some small changes in webpack config and maybe places like gitignore (sorry, forget lots of what's done here atm)